addons/google_calendar/utils/google_event.py

```diff
@@ -193,3 +193,12 @@ def _get_model(self, env):
         if all(not e.is_recurrence() for e in self):
             return env['calendar.event']
         raise TypeError("Mixing Google events and Google recurrences")
+
+    def get_meeting_url(self):
+        if not self.conferenceData:
+            return False
+        video_meeting = next(filter(lambda entryPoints: entryPoints['entryPointType'] == 'video', self.conferenceData['entryPoints']), None)
+        return video_meeting['uri'] if video_meeting else False
+
+    def is_show_has_available(self):
```

Comment:
def is_available() ?
is_show_has_available is not a correct name and is_show_as_available is quite long.

Reply:
@arj-odoo Thanks for the review!
You right :sweat_smile: I just updated the PR according to this